counterparty-lib/counterpartylib/lib/kickstart/blocks_parser.py

```diff
@@ -53,6 +52,7 @@ def fetch_blocks(bitcoind_dir, db_path, queue, first_block_index, parser_config)
         cursor.close()
         db.close()
 
+        parser = BlockchainParser(bitcoind_dir)
```

Comment:
For some reason, this call has to occur either before `aspw.Connection() ... db.close()` block or after. It cannot be in between or else I get this error:

```
[OK] Starting blocks parser from block 410339...
[=== ] Starting...src/tcmalloc.cc:304] Attempt to free invalid pointer 0x11896d200 
```